src/main/resources/config.properties

```diff
@@ -12,6 +12,7 @@ Builder.matlab.test.support.error = To run tests with the Jenkins plugin, use MA
 builder.matlab.automatictestoption.display.name = Automatic
 builder.matlab.customcommandoption.display.name = Custom
 Releaseinfo.matlab.version.not.found.error = Error finding MATLAB release for given MATLAB root. Verify MATLAB root path.
+matlab.not.found.error = Error finding MATLAB at given location. Verify MATLAB root path.
```

Comment:
Can we use this text?

Unable to launch MATLAB from the specified location. Verify the path to MATLAB root folder.

Reply:
Done.